audioplayer.cpp

```diff
@@ -0,0 +1,99 @@
+#include "audioplayer.h"
+
+vector<string> getFileNames(const string& folder) {
```

Comment:
Is `getAvailableTracks` or `getAvailableTrackFileNames` better?

Reply:
Btw you should make this function a method of audioplayer.

Reply:
done